script.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable linebreak-style */
```

Comment:
- [ ] Please don't disable linters it is not good practice. Linters are there to help you write a better code